syntax = "proto3";

import "google/protobuf/any.proto";

message SessionCommand {
    google.protobuf.Any command = 1;
}

message CommandPing {}

message CommandGetDatabase {}
message CommandGetImageLinks {}

message CommandUserIdentification {
    string name = 1;
}
